### ISSUE TITLE
Neural Network 

### ISSUE THREAD

**Issue by RJDay** [RJDay/DatabaseNetwork#2]
Set up neural network using C programming that will interact with the dataset. The dataset that is collected using the database will act as training data for the network.  

- [ ] Take database file and change into CSV file 
- [ ] Normalized data in the CSV file 
- [ ] Determine type of neural network needed to produce desired output 
- [ ] Determine the number of input and output neurons needed for the network
